src/AdvancedHMC.jl

```diff
@@ -139,9 +143,21 @@ include("sampler.jl")
 export sample
 
 include("abstractmcmc.jl")
-export DifferentiableDensityModel
 
-include("contrib/ad.jl")
+Hamiltonian(metric::AbstractMetric, ℓ::LogDensityModel) = Hamiltonian(
+    metric,
+    Base.Fix1(LogDensityProblems.logdensity, ℓ.logdensity),
+    Base.Fix1(LogDensityProblems.logdensity_and_gradient, ℓ.logdensity),
+)
+function Hamiltonian(metric::AbstractMetric, ℓπ::LogDensityModel, kind::Union{Symbol,Val})
+    ℓ = LogDensityModel(LogDensityProblemsAD.ADgradient(kind, ℓπ.logdensity))
```

Comment:
Forward keyword arguments?
```suggestion
function Hamiltonian(metric::AbstractMetric, ℓπ::LogDensityModel, kind::Union{Symbol,Val}; kwargs...)
    ℓ = LogDensityModel(LogDensityProblemsAD.ADgradient(kind, ℓπ.logdensity; kwargs...))
```